help/md/srGelmanRubin.md

```diff
@@ -1,9 +1,54 @@
 ## name
 srGelmanRubin
 ## title
+Gelman–Rubin stopping rule
+
 ## description
+Allow an MCMC run to terminate once the specified criterion has been met.
+The Gelman–Rubin rule compares the variance between chains with the variance within chains; its value tends to unity (1) as runs converge.
+
 ## details
+When passed as an element of the `StoppingRules[]` argument to `mcmc.run()` or `mcmcmc.run()`, the run will terminate once *all* specified convergence rules ([`srGelmanRubin()`], [`srGeweke()`], [`srMinESS()`], [`srStationarity()`]) have been fulfilled; or once *any* threshold rules ([`srMaxTime()`], [`srMaxIteration()`]) are met.
```

Comment:
OK - removed from the individual SR pages - on reflection I suppose that this is describing the behaviour of mcmc rather than the rule itself, and this is clear from the example.